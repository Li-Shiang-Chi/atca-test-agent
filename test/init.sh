



sudo mmsh createcluster test_c primary 85 127.0.0.1
sudo mmsh addnode test_c backup 192.168.1.101 86
sudo mmsh addnode test_c slave 192.168.1.102 87
