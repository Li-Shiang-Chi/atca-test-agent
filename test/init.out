2771
2722
create cluster success
add node success
3198
3225
create cluster success
add node success
